2022-08-07-qubes-os-summit-history.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Qubes OS Summit: History from organizer's perspective"
+categories: talks
```

Comment:
Please add an "author" field, like this:

```
author: <name>
```

`<name>` can be `3mdeb`, a comma-separated list of your personal names, or whatever else is appropriate.

Reply:
```suggestion
categories: talks
author: Michał Żygowski
```

Reply:
Added